src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/DependencyAnalysis/Target_Wasm/MetadataManager.Wasm.cs

```diff
@@ -86,6 +86,18 @@ private static bool IsPossibleDelegateTarget(NodeFactory factory, IWasmMethodCod
     public partial class ObjectDataInterner
     {
         public static ObjectDataInterner NullWithTracking { get; } =
-            new ObjectDataInterner() { _symbolRemapping = new() { [new ExternFunctionSymbolNode("")] = null } };
+            new CanFoldAlwaysObjectDataInterner() { _symbolRemapping = new() { [new ExternFunctionSymbolNode("")] = null } };
+
+        public sealed class CanFoldAlwaysObjectDataInterner : ObjectDataInterner
+        {
+            public CanFoldAlwaysObjectDataInterner() : base(genericsOnly: false)
+            {
+            }
+
+            public override bool CanFold(MethodDesc method)
```

Comment:
Do we need this override? The way I read https://github.com/yowl/runtimelab/blob/49b2b1185972adb45be1496035bb34a49d6a6fa7/src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/ObjectDataInterner.cs#L33-L34, it'll return `true` always with `_genericsOnly == false`.

Reply:
No, removed.